roles/federation-demo/tasks/postgres.yml

```diff
@@ -0,0 +1,47 @@
+---
+- name: Make sure psycopg2 is installed
+  command: pip install psycopg2==2.7.5 --ignore-installed
```

Comment:
You can also use the [pip ansible module](https://docs.ansible.com/ansible/2.9/modules/pip_module.html).

----------------------
References :
 
https://github.com/rciam/rciam-deploy/blob/a4e475a63833748108288f6b1fdc41b918001003/roles/fedreg-agent/tasks/deploy.yml#L16-L38

https://github.com/rciam/rciam-deploy/blob/a4e475a63833748108288f6b1fdc41b918001003/roles/fedreg-agent/tasks/configure-common.yml#L18-L38